src/ext/handlersToResp.c

```diff
@@ -930,6 +907,12 @@ static RdbRes toRespRestoreFragEnd(RdbParser *p, void *userData) {
 
 _LIBRDB_API RdbxToResp *RDBX_createHandlersToResp(RdbParser *p, RdbxToRespConf *conf) {
     RdbxToResp *ctx;
+    static int crcInitalized = 0;
+
+    if (!crcInitalized) {
+        ++crcInitalized;
+        crc64_init();
+    }
```

Comment:
isn't this dangerous when the lib is used from multiple threads (like we discussed in the past)?

Reply:
To my best understanding, it is generally safe for two different threads to write the same value to a variable (int this case it is a table) as long as there are no race conditions or other synchronization issues involved. Writing the same value(s) to a variable simultaneously won't cause data corruption or unexpected behavior by itself. 

If you still prefer to synchronize it, I can modify it to :

```
    static int crcInitalized = 0;
    static pthread_mutex_t mutex = PTHREAD_MUTEX_INITIALIZER;
    if (unlikely(!crcInitalized)) {
        /* If not init, acquire a lock to prevent concurrent init */
        pthread_mutex_lock(&mutex);
        if (!crcInitalized) {
            crc64_init();
            crcInitalized = 1;
       }
       pthread_mutex_unlock(&mutex);
    }
```

Reply:
It's probably safe to write to a variable but it could be that one thread concluded it can continue and the other thread re-init the table.
I'm afk and I don't know if re-init the table starts with zeroing or just overwrite, if it starts with zeroing your suggested solution isn't enough (thr mutex should wrap the initial flag check too

Reply:
Ohh I see the commit you made solves that

Reply:
You are right, there is a bug. 
I should have first call crc64_init() and only then set the flag and then claim my point :)
crc64_init doesn't zero the table. Just a set of assignments to the table.
In any way I pushed crc64_init_thread_safe() wrapper.